### PR TITLE
Fix #221 Remove google apis rs support

### LINKED ISSUE #221: Remove google-apis-rs support
Fully remove the google-apis-rs support code. This'll be a nice cleanup to the codebase

### PR DESCRIPTION
**Pull request by fzzzy**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -76,6 +76,5 @@ google-spanner1 = "1.0.10"
 yup-oauth2 = "^1.0"
 
 [features]
-default = ["google_grpc"]
+default = []
```
Comment (jrconlin) [inline Cargo.toml]: Drop?

File: src/db/spanner/support.rs [hunk 2]
```diff
@@ -79,7 +54,6 @@ pub enum SpannerType {
     Struct,
 }
 
-#[cfg(feature = "google_grpc")]
 impl Into<googleapis_raw::spanner::v1::type_pb::Type> for SpannerType {
```
Comment (pjenvey) [inline src/db/spanner/support.rs]: Not terribly important but we can also optionally kill the SpannerType enum. It's only for supporting both spanner libs simultaneously.

**Comment by pjenvey** [comment]
Also I noted some ideas for improvements of our support module in #260 (now that we only need to support the 1 client lib)

**Comment by fzzzy** [comment]
Thanks for the great suggestions @pjenvey

File: src/db/spanner/support.rs (hunk 2 above)
Comment (fzzzy) [reply src/db/spanner/support.rs]: Looks like this is a bigger change, so I opened a new bug for it:  #261.

File: Cargo.toml [hunk 3]
```diff
@@ -30,8 +30,8 @@ docopt = "1.1.0"
 env_logger = "0.6"
 failure = "0.1.5"
 futures = "0.1.28"
-googleapis-raw = { version = "0", path = "../mozilla-rust-sdk/googleapis-raw", optional = true }
-grpcio = { version = "0.5.0-alpha.1", optional = true }
+googleapis-raw = { version = "0", path = "../mozilla-rust-sdk/googleapis-raw" }
+grpcio = { version = "0.5.0-alpha.1" }
 lazy_static = "1.4.0"
 # XXX: switch to this line google-apis-rs support
```
[the remark this answers is not in the document]
Comment (fzzzy) [reply Cargo.toml]: Ok, done.

File: Cargo.toml (hunk 1 above)
Comment (fzzzy) [reply Cargo.toml]: Done

File: src/settings.rs [hunk 4]
```diff
@@ -134,11 +134,7 @@ impl Settings {
         let db = Url::parse(&self.database_url)
             .map(|url| url.scheme().to_owned())
             .unwrap_or_else(|_| "<invalid db>".to_owned());
-        let features = if cfg!(feature = "google_grpc") {
-            "+google_grpc"
-        } else {
-            ""
-        };
+        let features = "+google_grpc";
```
[the remark this answers is not in the document]
Comment (fzzzy) [reply src/settings.rs]: Done

File: src/db/spanner/manager.rs [hunk 5]
```diff
@@ -30,20 +22,12 @@ impl SpannerConnectionManager {
 }
 
 pub struct SpannerSession {
-    #[cfg(feature = "google_grpc")]
     pub client: googleapis_raw::spanner::v1::spanner_grpc::SpannerClient,
```
[the remark this answers is not in the document]
Comment (fzzzy) [reply src/db/spanner/manager.rs]: Done

**Pull request merged**